lib/gems/pending/util/miq_object_storage.rb

```diff
@@ -15,4 +19,33 @@ def initialize(settings)
   def logger
     @logger ||= $log.nil? ? :: Logger.new(STDOUT) : $log
   end
+
+  private
+
+  DONE_READING = ""
+  def read_single_chunk(chunksize = DEFAULT_CHUNKSIZE)
+    @buf_left   ||= byte_count
+    return DONE_READING unless @buf_left.nil? || @buf_left.positive?
+     cur_readsize  = if (@buf_left.nil? || @buf_left - chunksize >= 0)
```

Comment:
I think the spacing of these next size lines needs to be corrected by one space back.

Reply:
Fixed.